### PR TITLE
Add magma to itemtype damage cause comparator

### PR DESCRIPTION
**Pull request by TPGamesNL**
### Description
Adds magma to the itemtype - damage cause comparator

---
**Target Minecraft Versions:** any
**Requirements:** none
**Related Issues:** #4203

### CODE REVIEW

File: src/main/java/ch/njol/skript/classes/data/DefaultComparators.java [hunk 1]
```diff
@@ -464,18 +464,21 @@ public boolean supportsOrdering() {
 		ItemType lava = Aliases.javaItemType("lava");
 		Comparators.registerComparator(DamageCause.class, ItemType.class, new Comparator<DamageCause, ItemType>() {
 			@Override
-			public Relation compare(final DamageCause dc, final ItemType t) {
+			public Relation compare(DamageCause dc, ItemType t) {
```
Comment (bluelhf) [inline src/main/java/ch/njol/skript/classes/data/DefaultComparators.java:467]: Are these supposed to be non-final? You aren't changing them, so it seems strange that you got rid of the `final`s here

Comment (TPGamesNL) [reply src/main/java/ch/njol/skript/classes/data/DefaultComparators.java:467]: Skript's code conventions say not to use the final modifier on local variables and parameters: 
image

A lot of Skript's code is from before these code conventions existed, so whenever I modify some code in a PR, I also reformat the code if needed

Comment (bluelhf) [reply src/main/java/ch/njol/skript/classes/data/DefaultComparators.java:467]: Ah, that makes sense.

**Review by bluelhf — APPROVED** [review]
Looks good!

**Review by APickledWalrus — APPROVED** [review]
image

**Pull request merged**